### PR TITLE
docs(README): add repo README and yolov7-stomata README

### PR DESCRIPTION
**Pull request by heiruwu**
Because

- Provide guides and information on how to serve our sample models

This commit

- add repo README
- add yolov7-stomata README

the reaming models' README will be added soon

### CODE REVIEW

File: yolov7-stomata/README.md [hunk 1]
```diff
@@ -1,11 +1,90 @@
...
+                                         'left': 261,
+                                         'top': 336,
+                                         'width': 59},
+                        'category': 'outer_line',
+                        'rle': '11,10,19,17,13,19,11,20,10,22,9,22,8,23,6,25,5,26,5,26,4,27,4,28,3,28,3,28,3,28,3,15,4,9,3,14,6,8,3,14,6,8,3,14,6,8,3,14,6,8,3,14,6,9,2,14,6,9,2,14,6,9,2,14,6,9,2,14,6,9,2,14,6,9,2,15,4,10,2,29,2,29,2
+,29,2,29,2,28,3,28,3,28,3,28,3,28,3,28,3,28,3,28,3,28,3,28,3,28,3,27,4,27,4,27,4,27,4,27,4,26,5,25,6,23,8,22,9,21,10,21,11,19,13,17,16,13,21,7,78',
+                        'score': 0.8836774826049805}
+                        ...
+                        ...
+                        ...]}}]
```
Comment (GeorgeWilliamStrong) [inline yolov7-stomata/README.md:84]:
```suggestion
[
    {
        "data": {
            "objects": [
                {
                    "bounding-box": {
                        "height": 40,
                        "left": 306,
                        "top": 502,
                        "width": 67
                    },
                    "category": "outer_line",
                    "rle": "137,13,23,19,19,22,17,24,15,25,14,27,13,27,13,27,12,29,11,29,11,30,10,31,9,31,9,32,8,33,7,15,3,15,7,15,4,14,7,14,6,13,7,14,6,13,7,14,5,14,7,15,4,14,7,15,4,14,7,16,2,15,7,16,2,15,7,33,7,33,7,33,7,33,7,33,7,33,7,33,7,10,1,22,7,9,3,21,7,8,4,21,7,8,4,21,7,7,6,20,7,7,6,20,7,7,6,20,7,7,6,20,7,7,6,20,7,7,6,19,8,7,6,19,8,7,6,18,9,8,5,17,10,8,4,17,11,8,4,17,11,9,2,17,12,28,12,28,12,27,13,27,13,27,13,26,14,25,15,24,16,23,17,22,18,21,19,20,20,20,21,18,23,15,27,10,59",
                    "score": 0.9110509753227234
                },
                {
                    "bounding-box": {
                        "height": 35,
                        "left": 93,
                        "top": 26,
                        "width": 61
                    },
                    "category": "outer_line",
                    "rle": "117,13,20,18,15,21,14,22,12,23,11,25,9,27,7,29,5,31,3,32,3,33,2,33,2,33,2,33,2,15,3,15,2,14,5,14,2,14,6,13,2,14,6,13,2,13,7,13,2,13,7,13,2,13,7,13,2,13,7,13,2,13,7,13,2,13,7,13,2,13,7,13,2,14,6,13,2,14,6,13,2,14,6,13,2,14,5,14,2,14,5,14,2,14,5,14,2,15,3,15,2,16,2,15,2,33,2,33,2,33,2,33,2,33,2,33,2,33,2,33,2,32,3,31,4,31,4,30,5,29,6,29,6,28,8,26,9,26,10,24,12,23,13,21,15,19,17,17,20,12,25,9,49",
                    "score": 0.8888306021690369
                },
                {
                    "bounding-box": {
                        "height": 31,
                        "left": 261,
                        "top": 336,
                        "width": 59
                    },
                    "category": "outer_line",
                    "rle": "11,10,19,17,13,19,11,20,10,22,9,22,8,23,6,25,5,26,5,26,4,27,4,28,3,28,3,28,3,28,3,15,4,9,3,14,6,8,3,14,6,8,3,14,6,8,3,14,6,9,2,14,6,9,2,14,6,9,2,14,6,9,2,14,6,9,2,14,6,9,2,15,4,10,2,29,2,29,2,29,2,29,2,28,3,28,3,28,3,28,3,28,3,28,3,28,3,28,3,28,3,28,3,28,3,27,4,27,4,27,4,27,4,27,4,26,5,25,6,23,8,22,9,21,10,21,11,19,13,17,16,13,21,7,78",
                    "score": 0.8836774826049805
                }
                ...
                ...
                ...
            ]
        }
    }
]
```

File: yolov7-stomata/README.md [hunk 2]
```diff
@@ -1,11 +1,90 @@
...
+| Model Version | Instill-Core Version | Python-SDK Version |
+| ------------- | -------------------- | ------------------ |
+| v0.0.1        | <=v0.39.0-beta       | <=0.11.0, >=0.10.2 |
+| v0.1.0        | >v0.39.0-beta        | >0.11.0            |
+
+> **Note:** Always ensure that you are using compatible versions to avoid unexpected issues.
+
+## 🚀 Preparation
+
+Follow [this](./README.md) guide to get your custom model up and running! But before you do that, please read through the following sections to have all the necessary files ready.
```
Comment (GeorgeWilliamStrong) [inline yolov7-stomata/README.md:24]:
```suggestion
Follow [this](../README.md) guide to get your custom model up and running! But before you do that, please read through the following sections to have all the necessary files ready.
```

File: yolov7-stomata/README.md [hunk 3]
```diff
@@ -1,11 +1,90 @@
...
 
-# YoloV7 for Stomata identification
+### Install SDK
 
-Instill Model compatible implementation of [StomaVision](https://github.com/YaoChengLab/StomaVision) model.
+- Install the compatible python-sdk version according to the compatibility matrix
```
Comment (GeorgeWilliamStrong) [inline yolov7-stomata/README.md:28]:
```suggestion
- Install the compatible [`python-sdk`](https://github.com/instill-ai/python-sdk) version according to the compatibility matrix
```

File: yolov7-stomata/README.md [hunk 4]
```diff
@@ -1,11 +1,90 @@
...
+
+[yolov7-stomata](https://github.com/YaoChengLab/StomaVision) is a machine learning model designed for stomata detection and segmentation, check out the link to lean more.
+
+| Task Type                                                                                  | Description                                                                                         |
+| ------------------------------------------------------------------------------------------ | --------------------------------------------------------------------------------------------------- |
+| [Instance Segmentation](https://www.instill.tech/docs/model/ai-task#instance-segmentation) | A vision task to detect and delineate multiple objects of pre-defined categories in an input image. |
+
+## 🔄 Compatibility Matrix
+
+To ensure smooth integration, please refer to the compatibility matrix below. It outlines the compatible versions of the model, `instill-core`, and the `python-sdk`.
```
Comment (GeorgeWilliamStrong) [inline yolov7-stomata/README.md:13]:
```suggestion
To ensure smooth integration, please refer to the compatibility matrix below. It outlines the compatible versions of the model, [`instill-core`](https://github.com/instill-ai/instill-core), and the [`python-sdk`](https://github.com/instill-ai/python-sdk).
```

File: yolov7-stomata/README.md [hunk 5]
```diff
@@ -1,11 +1,90 @@
...
+| v0.1.0        | >v0.39.0-beta        | >0.11.0            |
+
+> **Note:** Always ensure that you are using compatible versions to avoid unexpected issues.
+
+## 🚀 Preparation
+
+Follow [this](./README.md) guide to get your custom model up and running! But before you do that, please read through the following sections to have all the necessary files ready.
 
-# YoloV7 for Stomata identification
+### Install SDK
```
Comment (GeorgeWilliamStrong) [inline yolov7-stomata/README.md:26]:
```suggestion
### Install Python SDK
```

File: yolov7-stomata/README.md [hunk 6]
```diff
@@ -1,11 +1,90 @@
...
+
+get the model weight by executing the following command.
+
+```bash
+curl -o model.pt https://artifacts.instill.tech/model/yolov7-stomata/model.pt
+```
+
+## Test model image
+
+After you've built the model image and before pushing the model onto any Instill-Core instance, you can test if the model can be successfully run locally first, by running the following command:
```
Comment (GeorgeWilliamStrong) [inline yolov7-stomata/README.md:44]:
```suggestion
After you've built the model image, and before pushing the model onto any **Instill Core** instance, you can test if the model can be successfully run locally first, by running the following command:
```

File: yolov7-stomata/README.md [hunk 7]
```diff
@@ -1,11 +1,90 @@
...
+
+```bash
+instill run admin/yolov7-stomata -i '{"image-url": "https://microscopyofnature.com/sites/default/files/2022-03/Mais-stomata-ZW10.jpg", "type": "image-url"}'
+```
+
+And you should get a output like this
```
Comment (GeorgeWilliamStrong) [inline yolov7-stomata/README.md:50]:
```suggestion
A successful response will return a similar output to that shown below.
```

File: README.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
+# Instill AI Models
+
+Welcome to our Model Repository! This repository houses a collection of machine learning models designed to perform various AI tasks, and be seamlessly served on [Instill Core](https://github.com/instill-ai/instill-core) or [Instill Cloud](www.instill.tech).
```
Comment (GeorgeWilliamStrong) [inline README.md:3]:
```suggestion
Welcome to our Model Repository! This repository houses a collection of machine learning models designed to perform various AI tasks. They are all prepared and ready to be seamlessly served on [Instill Core](https://github.com/instill-ai/instill-core) or [Instill Cloud](www.instill.tech) via our MLOps/LLMOps platform [Instill Model](https://www.instill.tech/docs/model/introduction).
```

File: README.md [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+
+## Getting Started
+
+We leverage Instill Core to provide a seamless experience for serving models. Follow the steps below to quickly get started:
+
+### 1. Instill Core or Instill Cloud
+
+#### Self-host Instill-Core
+
+- Follow this [quickstart guide](https://www.instill.tech/docs/quickstart) to get it up and running.
```
Comment (GeorgeWilliamStrong) [inline README.md:33]:
```suggestion
Follow [this section](https://www.instill.tech/docs/quickstart#-instill-core) of our quick start guide to get it up and running with self-hosting **Instill Core** on a local or remote instance.
```

File: README.md [hunk 10]
```diff
@@ -0,0 +1,69 @@
...
+| [yolov7-stomata](./yolov7-stomata/README.md)                   | Instance Segmentation | yolov7-stomata is designed for stomata detection and segmentation.                                                     |
+| [zephyr-7b](./zephyr-7b/README.md)                             | Chat                  | zephyr-7b is a series of language models that are trained to act as helpful.assistants.                                |
+
+## Getting Started
+
+We leverage Instill Core to provide a seamless experience for serving models. Follow the steps below to quickly get started:
+
+### 1. Instill Core or Instill Cloud
+
+#### Self-host Instill-Core
```
Comment (GeorgeWilliamStrong) [inline README.md:31]:
```suggestion
#### Self-host 🔮 Instill Core
```

File: README.md [hunk 11]
```diff
@@ -0,0 +1,69 @@
...
+
+- Follow this [quickstart guide](https://www.instill.tech/docs/quickstart) to get it up and running.
+
+#### Instill Cloud
```
Comment (GeorgeWilliamStrong) [inline README.md:35]:
```suggestion
#### ☁️ Instill Cloud
```

File: README.md [hunk 12]
```diff
@@ -0,0 +1,69 @@
...
+
+- Go to our [website](https://www.instill.tech/) and check out what we have to offer.
```
Comment (GeorgeWilliamStrong) [inline README.md:37]:
```suggestion
Follow [this section](https://www.instill.tech/docs/quickstart#%EF%B8%8F-instill-cloud) of our quick start guide to get it up and running on **Instill Cloud**, our fully managed public cloud service that provides you with access to all the features of **Instill Core** without the burden of infrastructure management.
```

File: README.md [hunk 13]
```diff
@@ -0,0 +1,69 @@
...
+
+### 2. Create a Namespace
```
Comment (GeorgeWilliamStrong) [inline README.md:39]:
```suggestion
### 2. Create a model namespace
```

File: README.md [hunk 14]
```diff
@@ -0,0 +1,69 @@
...
+
+To create a namespace, follow the steps in [Create a Namespace Documentation](https://www.instill.tech/docs/model/create/namespace).
```
Comment (GeorgeWilliamStrong) [inline README.md:41]:
```suggestion
To create a model namespace, follow the steps on the [Create Namespace](https://www.instill.tech/docs/model/create/namespace) page.
```

File: README.md [hunk 15]
```diff
@@ -0,0 +1,69 @@
...
+
+### 3. Prepare your model
+
+Find the model you want to serve and download the desired version folder, also check out the particular model folder README to obtain other necessary files and model weights or perform necessary steps.
```
Comment (GeorgeWilliamStrong) [inline README.md:45]:
```suggestion
Find the model you want to serve and download the desired version folder. Also, make sure to check out the particular model folder README to obtain other necessary files, model weights or perform any additional required steps.
```

File: README.md [hunk 16]
```diff
@@ -0,0 +1,69 @@
...
+
+### 4. Build your model
+
+Follow the steps in [Build Your Model Documentation](https://www.instill.tech/docs/model/create/build), and remember to install the python-sdk version according to the compatibility matrix in each model's README.
```
Comment (GeorgeWilliamStrong) [inline README.md:49]:
```suggestion
Follow the steps on the [Build Model Image](https://www.instill.tech/docs/model/create/build) page, and remember to install the `python-sdk` version according to the compatibility matrix in each model's README.
```

File: README.md [hunk 17]
```diff
@@ -0,0 +1,69 @@
...
+
+### 5. Push and deploy your model
+
+Follow the steps in [Push Your Model Documentation](https://www.instill.tech/docs/model/create/push) to deploy the model.
```
Comment (GeorgeWilliamStrong) [inline README.md:53]:
```suggestion
Follow the steps on the [Push Model Image](https://www.instill.tech/docs/model/create/push) page to deploy the model to your choice of **Instill Core** or **Instill Cloud**.
```

File: README.md [hunk 18]
```diff
@@ -0,0 +1,69 @@
...
+
+## Implement your own custom model
+
+Follow the steps in [Prepare Model](https://www.instill.tech/docs/model/create/prepare) to see how to implement your own custom model that can be served on Instill Core and Instill Cloud!
```
Comment (GeorgeWilliamStrong) [inline README.md:57]:
```suggestion
Follow the steps on the [Prepare Model](https://www.instill.tech/docs/model/create/prepare) page to see how to implement your own custom model that can be served on **Instill Core** and **Instill Cloud**! You can also checkout the [step-by-step tutorial](https://www.instill.tech/blog/model-serving-on-instill-core) which walks you through the process of serving your own custom model on **Instill Core**.
```

**Review by GeorgeWilliamStrong — COMMENTED** [review]
I have left a bunch of suggestions for you to consider. Looks great!

**Comment by heiruwu** [comment]
@GeorgeWilliamStrong thanks for all the great suggestions!

**Pull request merged**